src/dvc_objects/fs/implementations/gs.py

```diff
@@ -23,3 +23,9 @@ def fs(self):
         from gcsfs import GCSFileSystem
 
         return GCSFileSystem(**self.fs_args)
+
+    @classmethod
+    def _strip_protocol(cls, path: str) -> str:
+        from fsspec.utils import infer_storage_options
+
+        return infer_storage_options(path)["path"]
```

Comment:
I'm not sure I follow. In https://github.com/iterative/dvc/pull/7655/files you specifically wrote that we don't need to strip anything and here we are stripping the url down to the path. So what should we do and why?

Reply:
As mentioned in iterative/dvc#7655, only this `_strip_protocol` change is required to fix the import-url issue

Reply:
Got a bit confused by things moving around. Should be fine now, the iterative/dvc#7655 can be closed